sphinxcontrib/redoc.py

```diff
@@ -134,7 +135,7 @@ def assets(app, exception):
     # need to either ensure its existence here or do not try to copy  assets
     # in case of failure.
     if not exception:
-        copyfile(
+        copy_asset(
             os.path.join(_HERE, 'redoc.js'),
             os.path.join(app.builder.outdir, '_static', 'redoc.js'))
```

Comment:
According to [Sphinx code](https://github.com/sphinx-doc/sphinx/blob/574f448b634318d7e786fc45b8b42608b2b888ca/sphinx/util/fileutil.py#L66), the destination is a path to a destination directory. Can you please remove `'redoc.js'` at the end and leave only `os.path.join(app.builder.outdir, '_static')`?

Reply:
done